core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostService.scala

```diff
@@ -13,25 +13,27 @@ import scala.collection.JavaConverters._
 import scala.concurrent.{ExecutionContext, Future}
 
 object SubmissionCostService {
```

Comment:
None of the logic in this has changed, it just now defaults to the previous tableName if one was not specified